src/main/resources/db/migration/V202409161350__vaccineToPet_table.sql

```diff
@@ -1,4 +1,4 @@
-create table vaccineToPet
+create table vaccine_to_pet
```

Comment:
This change should not be in this PR (I'm not sure it will even work since the script was already ran by flyway).

Reply:
fixed